regression/python/try-fail/main.py

```diff
@@ -0,0 +1,3 @@
+from jira import JIRA  # this imported module contains try statements
```

Comment:
Where is the try-statement in this code? Did I miss something?

Reply:
The issue is that for every import of "module," we generate a corresponding "module.json." In the case of an "import jira," the interpreter resolves this to the file <path>/jira/__init__.py. Thus, we were generating an __init__.json, while the Python converter was expecting a jira.json. This import scenario exercises that condition, and I added this check to demonstrate that the module was imported; however, we do not support the try block contained in the code.